aci/resource_aci_rtctrlattrp.go

```diff
@@ -94,9 +97,19 @@ func resourceAciActionRuleProfile() *schema.Resource {
 				ConflictsWith: []string{"set_route_tag"},
 				Description:   "Invalid Configuration Set nexthop unchanged action cannot be configured along with set route tag action under the set action rule profile.",
 			},
+			"saspath_prepend_last_as": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validateIntRange(1, 10),
+			},
+			"saspath_prepend_asn": {
+				Type:     schema.TypeMap,
```

Comment:
From what I can see multiple ASN and Order can be configured so I would transform this into a TypeSet